scripts/tests/systemd/firezone-client.service

```diff
@@ -3,7 +3,8 @@ Description=Firezone Client
 
 [Service]
 AmbientCapabilities=CAP_NET_ADMIN
-CapabilityBoundingSet=CAP_NET_ADMIN
+# TODO: Get rid of `CAP_CHOWN` here by asking systemd to make our runtime dir on our behalf
+CapabilityBoundingSet=CAP_CHOWN CAP_NET_ADMIN
```

Comment:
This was some kind of odd merge conflict. I started using `chown` in one PR and then locked down the service file in another. Turns out chown has its own entire cap you have to enable.